src/js/utils.js

```diff
@@ -17,12 +17,28 @@ exports.parseQueryParams = () => {
 /** Creates an object based on a form's current input values. */
 exports.parseForm = form => {
   const formData = new FormData(form);
-  return Array.from(formData.entries()).reduce((acc, [key, value]) => {
-    if (!acc[key]) {
-      acc[key] = [];
+  if (!formData.entries) {
```

Comment:
Minor, but you could check `if (FormData && FormData.prototype.entries)` to avoid constructing FormData which might not exist or exists but then the instance isn’t used.

Reply:
Yes that makes it cleaner, done.